src/controllers/SoundController.ts

```diff
@@ -0,0 +1,43 @@
+const sound0 = require('../../static/sounds/0.mp3');
+const sound1 = require('../../static/sounds/1.mp3');
+const sound2 = require('../../static/sounds/2.mp3');
+const soundCapture = require('../../static/sounds/capture.mp3');
+
+import { SoundTypes } from '../enums/SoundTypes';
+
+class SoundController {
+    private lastPlayedMoveSound: number = 0;
+
+    public makeSound(soundType: SoundTypes) {
+        switch (soundType) {
+            case SoundTypes.NORMAL_MOVE:
+                this.playNormalMoveSound();
+                break;
+            case SoundTypes.CAPTURING_MOVE:
+                this.playCapturingMoveSound();
+                break;
+            default:
+                break;
+        }
+    }
+
+    private playCapturingMoveSound(): void {
```

Comment:
Zastanawiam się czy potrzebujemy ten enum i metode powyżej. Wydaje mi się że moglibyśmy te dwie prywatne metody ustawić jako publiczne i usunąć całe makeSound. Wtedy składnia do zagrania dzwięku wyglądałaby w ten sposób `soundService.playCapturingMoveSound()` zamiast `soundService.makeSound(SoundTypes.CAPTURING_MOVE)` 

To pierwsze wydaje się bardziej obiektowym podejściem bo jako użytkownik tego serwisu dostaje gotowy sposób jak zagrać konkretną melodyjkę i nie muszę importować żadnego SoundTypes

Reply:
no można, można